Cargo.toml

```diff
@@ -3,46 +3,53 @@ resolver = "2"
 members = ["crates/*"]
 
 [workspace.dependencies]
-anyhow = "1"
-async-trait = "0.1"
+anyhow = "1.0.82"
+async-trait = "0.1.80"
+axum = "0.6"
 bigdecimal = "0.3"
```

Comment:
We cannot update `bigdecimal` so easily, they removed the `sqlx` immediate support, very likely it is solved by just converting it into a primitive, but it needs more research to really check this, otherwise we can make break our database. What's more, this crate update is really not that needed at the moment.